src/MSBuildBinLogQuery/Component/Build.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Concurrent;
+using Microsoft.Build.Framework;
+
+namespace Microsoft.Build.Logging.Query.Component
+{
+    public class Build : Component
+    {
+        public ConcurrentDictionary<int, Project> Projects { get; }
+        public ConcurrentBag<Project> OrderedProjects { get; }
```

Comment:
Does order make sense here? While targets within a project and tasks within a target have an observed order, projects can be entered multiple times, including even reentrantly. I think the only meaningful order on projects must be derived from the graph, or possibly from a list of ProjectStartEvents (but I don't know what you'd really get out of that).

Reply:
The order of `OrderedProjects` is derived from a list `ProjectStartEvents`. I added it just in case we need information of ordered projects. But I can drop it if that won't be useful.

Reply:
Done.